src/elements/button/common/mini-button-common.scss

```diff
@@ -1,6 +1,6 @@
 @use '../../core/styles' as sbb;
 
-$disabled: '[disabled], :disabled, [disabled-interactive], [data-disabled], [data-group-disabled]';
+$disabled: '[disabled], :disabled, [disabled-interactive], [data-disabled], :state(disabled)';
```

Comment:
do you know in which case the mini button receives :state(disabled)? maybe it's obsoloete?